docker-compose.dev.yaml

```diff
@@ -0,0 +1,40 @@
+version: "3"
+
+services:
```

Comment:
This is incomplete, below to the list of services:

```
.
.
.
.
  web:
    build:
      context: .
      dockerfile: ./Dockerfile.dev
    image: mediacms/mediacms:latest
    ports:
      - "80:80"
    volumes:
      - ./:/home/mediacms.io/mediacms/
    depends_on:
      redis:
        condition: service_healthy
      db:
        condition: service_healthy
  db:
    image: postgres
    volumes:
      - ../postgres_data:/var/lib/postgresql/data/
    restart: always
    environment:
      POSTGRES_USER: mediacms
      POSTGRES_PASSWORD: mediacms
      POSTGRES_DB: mediacms
    healthcheck:
      test: ["CMD-SHELL", "pg_isready -U mediacms"]
      interval: 10s
      timeout: 5s
      retries: 5
  redis:
    image: "redis:alpine"
    restart: always
    healthcheck:
      test: ["CMD", "redis-cli","ping"]
      interval: 30s
      timeout: 10s
      retries: 3
```

Reply:
Actually we don't need to write all of them again. If we simply run
`docker-compose -f docker-compose.yaml -f docker-compose.dev.yaml up` it will run docker-compose first and then overrite non-existant settings from docker-compose.dev and run them

Reply:
Good suggestion, I would use `extends` (https://docs.docker.com/compose/extends/#extending-services) instead of depending on the user getting the command line right

Also, take note:

* some services have been removed (in dev mode they will all run in the web container)
* the `Dockerfile.dev` is used to create the image for the web container in dev mode